src/glib/base/blobbs.cpp

```diff
@@ -451,8 +457,13 @@ void TMBlobBs::SaveMain() const {
 TMBlobBs::TMBlobBs(
  const TStr& BlobBsFNm, const TFAccess& _Access, const int& _MxSegLen):
   TBlobBs(), Access(_Access), MxSegLen(_MxSegLen),
-  NrFPath(), NrFMid(), SegV(), CurSegN(0){
+  NrFPath(), NrFMid(), SegV() {
   if (MxSegLen==-1){MxSegLen=MxBlobFLen;}
+  // initialize the hashtable of block sizes to first segment
+  TBlobBs::GenBlockLenV(BlockLenV);
+  for (uint N = 0; N < BlockLenV.Len(); N++) {
```

Comment:
Throws warning during compile time: `warning: comparison is always true due to limited range of data type`. Best to use `int` for `N` and not `uint`